client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/getway/ILocalGateway.kt

```diff
@@ -0,0 +1,18 @@
+package org.thechance.common.domain.getway
+
+
+interface ILocalGateway {
```

Comment:
Why local gateway???
I understand that it's like the repo,then one place contains the difference sources right?!! Or it's like the data source so we should to separate it? "((
I'm confused"((